registry/github_registry.go

```diff
@@ -38,6 +39,7 @@ type githubRegistry struct {
 	format         common.RegistryFormat
 	credentialName string
 	service        GithubRepositoryService
+	httpClient     *http.Client
```

Comment:
You're right. It's package private. However, when looking for a way to get at the client, I noted [this method](https://github.com/google/go-github/blob/master/github/github.go#L313). Do we need the client for anything other than Do?